core/types/transaction.go

```diff
@@ -628,6 +632,11 @@ func (tx *Transaction) FeeCurrency() *common.Address {
 	return copyAddressPtr(tx.inner.feeCurrency())
 }
 
+// MaxFeeInFeeCurrency returns the max fee in the fee_currency for celo denominated txs.
```

Comment:
The comment feels too much like the variable name. We can add some more information here.

Suggestion:
> MaxFeeInFeeCurrency is only used to guard against very quickly changing exchange rates. Txs must be discarded if MaxFeeInFeeCurrency is exceeded.

Reply:
Done in dbb2c5759723